apps/bfd-server/bfd-server-docs/data-dictionary/data/V1/0011_brnd_gnrc_cd.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": 11,
-    "name": "Brand Generic Code",
+    "name": "Brand Generic Indicator",
```

Comment:
This is my fault for suggesting edits to _data dictionary entries_ (plural). This change should not have any impact on the implementation in v1. Sorry about that!